.github/workflows/build.yml

```
@@ -86,6 +90,49 @@ jobs:
           path: /tmp/out/*
           if-no-files-found: error
 
+  bin-image:
+    runs-on: ubuntu-20.04
+    if: ${{ github.event_name != 'pull_request' && github.repository == 'docker/cli' }}
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v4
```

You might want the same fetch-depth input as build so correct ref is returned in https://github.com/docker/cli/blob/8812e0ad3ad786d6aa9f6289df4656f096d493a0/scripts/build/.variables#L13

```suggestion
        uses: actions/checkout@v4
        with:
          fetch-depth: 0
```

---

Oh my bad didn't catch you were setting `VERSION` env. Looks good then!

---

Yeah, I did set the `VERSION` explicitly, so we shouldn't need the whole history.